### PR TITLE
Math proofread

### PR DESCRIPTION
**Pull request by goldmermaid**
Includes:
1. Math code full chapter scan;
2. distributions.md text scan;
3. random-variables.md text scan.

### CODE REVIEW

File: chapter_appendix_math/random-variables.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ However, when we look closer, this does not match our question!  We wanted exact
...
 
-However, this does not solve anything!  We have just pushed the issue down one digit further.  Let us abstract a bit. Imagine we know the probability that the first $k$ digits match with $2.00000\ldots$ and we want to know the probability it matches for the first $k+1$ digits. It is fairly reasonable to assume that the $k+1$-st digit is essentially a random choice from the set $\{0, 1, 2, \ldots, 9\}$.  At least, I cannot conceive of a physically meaningful process which would force the number of micrometers away form the center to prefer to end in a $7$ vs a $3$.  
+However, this does not solve anything!  We have just pushed the issue down one digit further.  Let us abstract a bit. Imagine we know the probability that the first $k$ digits match with $2.00000\ldots$ and we want to know the probability it matches for the first $k+1$ digits. It is fairly reasonable to assume that the ${k+1}^{\mathrm{st}}$ digit is essentially a random choice from the set $\{0, 1, 2, \ldots, 9\}$.  At least, we cannot conceive of a physically meaningful process which would force the number of micrometers away form the center to prefer to end in a $7$ vs a $3$.
```
Comment (astonzhang) [inline chapter_appendix_math/random-variables.md]: mytodo, st to th

File: chapter_appendix_math/random-variables.md [hunk 2]
```diff
@@ -175,9 +175,9 @@ $$
 
 Thus we see that the mean is $a$.  This matches the intuition since $a$ is the location around which we centered our random variable.
 
-Because they are helpful, let us summarize a few properties:
```
Comment (astonzhang) [inline chapter_appendix_math/random-variables.md]: mytodo, colon

File: chapter_appendix_math/random-variables.md [hunk 3]
```diff
@@ -209,46 +209,46 @@ $$
 
 This result again makes sense.  The largest $p$ can be is $1/2$ which corresponds to picking $a-2$ or $a+2$ with a coin flip.  The variance of this being $4$ corresponds to the fact that both $a-2$ and $a+2$ are $2$ units away from the mean, and $2^2 = 4$.  On the other end of the spectrum, if $p=0$, this random variable always takes the value $0$ and so it has no variance at all.
 
-We will list a few properties of variances below.
```
Comment (astonzhang) [inline chapter_appendix_math/random-variables.md]: mytodo, remove the

File: chapter_appendix_math/random-variables.md [hunk 4]
```diff
@@ -397,24 +397,24 @@ $$
 
 for some function $p(x, y)$.  This is referred to as the joint density of $X$ and $Y$.  Similar properties are true for this as we saw in the single variable case. Namely:
 
-* $p(x, y) \ge 0$
-* $\int _ {\mathbb{R}^2} p(x, y) \;dx \;dy = 1$
+* $p(x, y) \ge 0$;
```
Comment (astonzhang) [inline chapter_appendix_math/random-variables.md]: mytodo,  punctuate within $$

File: chapter_appendix_math/statistics.md [hunk 5]
```diff
@@ -35,13 +35,13 @@ npx.set_np()
 epsilon = 0.1
 random.seed(8675309)
 xs = np.random.normal(loc=0, scale=1, size=(300, ))
-ys = [np.sum(np.exp(-(xs[0:i] - xs[i])**2/(2 * epsilon**2)) / 
-      np.sqrt(2*np.pi*epsilon**2))/len(xs) for i in range(len(xs))]
+ys = [np.sum(np.exp(-(xs[0:i] - xs[i])**2/(2 * epsilon**2)) /
```
Comment (astonzhang) [inline chapter_appendix_math/statistics.md]: mytodo, rm space

File: chapter_appendix_math/random-variables.md (hunk 2 above)
Comment (goldmermaid) [reply chapter_appendix_math/random-variables.md]: Gotcha

File: chapter_appendix_math/random-variables.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_appendix_math/random-variables.md]: https://english.stackexchange.com/questions/36512/k1th-or-k1st

**Pull request merged**